src/CreatePackage.ts

```diff
@@ -47,6 +51,20 @@ function createZipArchive(zip: AdmZip, folder: string, options: ZipOptions) {
     }
 }
 
+const buildFolder = 'production_modules';
```

Comment:
Lepsi specifikovat na zacatku souboru, protoze pouzivas i ve funkci nad.. a mozna by bylo lepsi pojmenovat jako productionModulesFolder -> build folder to puvodne mel byt, ale pak jsme to zmenili

Reply:
!